lib/hls/hls_parser.js

```diff
@@ -1219,6 +1220,12 @@ shaka.hls.HlsParser = class {
       const [width, height] = resolution ? resolution.split('x') : [null, null];
 
       const videoRange = tag.getAttributeValue('VIDEO-RANGE');
+      let videoLayout = tag.getAttributeValue('REQ-VIDEO-LAYOUT');
+      if (videoLayout == 'CH-STEREO,CH-MONO') {
```

Comment:
I might be missing a more detailed description, but from what I see in the HLS authoring spec:
https://developer.apple.com/documentation/http-live-streaming/hls-authoring-specification-for-apple-devices

> If stereo video content includes sections of monoscopic video content, then the REQ-VIDEO-LAYOUT attribute MUST include “CH-MONO” as one entry.

> All variants containing stereo video content MUST be marked with a REQ-VIDEO-LAYOUT attribute.

That suggests to me that if a `REQ-VIDEO-LAYOUT` is present with both `CH-MONO` and `CH-STEREO`, it probably represents a video stream that contains both monoscopic and stereoscopic video content.
However, I didn't see anything to suggest that the video layout listed first was the "primary" one, which is what these lines of code seem to be assuming. Is there a spec somewhere that says that?